### PR TITLE
Release 3.5.2

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.2](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.2)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.5.2 (2020-02-13)
+
+- Fixed the grid component to handle scroll events when the top scroll bar fails to render. [#100](https://github.com/blackbaud/skyux-grids/pull/100)
+- Fixed the grid component to properly show the top scroll bar when needed due to the window resizing. [#101](https://github.com/blackbaud/skyux-grids/pull/101)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the grid component to properly display the top scroll bar as needed when windows are resized.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Done

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.2](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.5.2)

**Pull request merged**